### DISCUSSION TITLE
为什么内置的表达式作用域只支持 x-reactions 中使用？

### DISCUSSION THREAD

**Discussion by liuweiGL** [alibaba/formily#1212] [Q&A 2.x] [answered]
## 问题
schema 里面的属性都是支持表达式的，但是只有 `x-reactions` 属性有内置表达式支持。我觉得 `$form` `$self` 这种通用的作用域应该对所有属性注入啊？

## 场景
表单页面有很多操作类的组件，详情模式是不需要展示的，所以希望使用 `{{ $form.editable }}`  来设置 schema 里面的 `visible`  属性：
![image](https://user-images.githubusercontent.com/21048878/114482521-4931c700-9c39-11eb-89fe-ac1154cbd57a.png)

**Comment by janryWang** [comment, marked as answer]
因为其他属性是无法做到响应式更新的，只适用于初次渲染，只有x-reactions支持响应式更新

**Comment by liuweiGL** [comment]
2.0.0-rc5 已经支持了，棒棒的，很多简单的业务关联不需要再使用 x-reactions 来处理了。
